### PR TITLE
Release 20211120

### PR DESCRIPTION
**Pull request by Realmonia**
### Summary

Changes for 20211120 release.

This release is related to CVE-2021-41190
New packages required in repo:
```
AL2 containerd-1.4.6-7.amzn2
AL2 docker-20.10.7-5.amzn2
AL1 containerd-1.4.6-7.11.amzn1
AL1 docker-20.10.7-5.76.amzn1
```

### Testing

New tests cover the changes: 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sharanyad — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 20211120
+
```
Comment (sparrc) [inline CHANGELOG.md:4]: we dont need any changelog entries here because none of these have changed from the previous release

Comment (sharanyad) [reply CHANGELOG.md:4]: al1?

Comment (sparrc) [reply CHANGELOG.md:4]: err I guess this AL1 ami name should actually be under 20211115, must have missed that last week

Comment (Realmonia) [reply CHANGELOG.md:4]: Oh, I will update this.

**Review by fenxiong — APPROVED** [review]

**Review by chienhanlin — APPROVED** [review]

**Pull request merged**